relayer/provider/provider.go

```diff
@@ -45,8 +46,10 @@ type LatestBlock struct {
 }
 
 type IBCHeader interface {
-	IBCHeaderIndicator()
 	Height() uint64
+	ConsensusState() ibcexported.ConsensusState
+	// require conversion implementation for third party chains
+	ToCosmosValidatorSet() (*tmtypes.ValidatorSet, error)
```

Comment:
Will there end up being a similar function for each `Provider` implementation in the future?

Reply:
Thanks for calling this out, I removed the need for this so it is dead code now.